### DISCUSSION TITLE
Early testing of the next release.

### DISCUSSION THREAD

**Discussion by danielkschneider** [inkstitch/inkstitch#1521] [General]
Hi, I would like to have a look at a development snapshot for 2 reasons: (1) Check the translation strings and (2) satisfy my curiosity.

Would any recent snapshot, e.g. https://github.com/inkstitch/inkstitch/releases/tag/dev-build-kaalleen-gcode-max-z be a good choice ?

If so, is it safe to use in production (I imagine so since I cannot image any machine breaking code that you would come up with. :=)

thanx ! Daniel

**Comment by kaalleen** [comment]
Hey there. Sorry, I didn't see your question. I somehow always miss what is written in the discussions.
In the mentioned branch I am afraid there aren't the newest translations included. Therefore I've updated this branch here: https://github.com/inkstitch/inkstitch/releases/tag/dev-build-kall-test

This branch has the exact same code as the main branch in its current state. The Linux version may have an issue in the params and the lettering extension and may not be able to display the font correctly (instead you'll see squares). This will need to be fixed. Also the linux installation may still be updated. Lex has been working on it. The macOS installation file is unsigned and needs to be opened with Ctrl+Click > Open instead of normal click. The Windows installation routine may complain about an untrusted certificate until we gain enough trust (through many more active installations). Click on "more" and "run anyway" to bypass this.

Thanks for taking the time to test development versions!!

**Reply by danielkschneider** [reply]
Hi Kalleen thanx (I forgot myself to check for an answer .... )

Thanx ! I will have a look at the windows version then. It did install using your installer and it does seem to work. I first will fix the shortcuts and then see how it behaves for what I usually do :)

I also have Linux for Beta/Release candidate testing once you need that :)
- daniel

**Reply by kaalleen** [reply]
Thanks. It's good to hear that the windows version works. This is about to be released now. Still, it would be very helpful to get some feedback on the updated installation routines and upcoming error messages. I have a feeling, that there will be some fixes necessary which we then can push into a follow up release.
